docs/reference/hooks/interchain-gas.mdx

```diff
@@ -113,3 +172,54 @@ If the `handle` call consumes more gas than quoted, the relayer will not submit
 
 </TabItem>
 </Tabs>
+
+## Using the Hyperlane Explorer for Debugging
+
+The Hyperlane Explorer is a powerful tool to debug cross-chain message issues, including gas payments and relayer behavior.
+
+### Key Features
+
+- **Message Status**: View the current status of your message (e.g., "Retry: GasPaymentRequirementNotMet").
+- **Gas Payment Details**: Check the gas amount paid (Origin IGP gas amount) and the amount required by the relayer.
+- **Simulate Calldata**: Use the "View calldata details" option to simulate transactions on tools like Tenderly.
+
+## Troubleshooting
+
+This section covers common issues developers encounter with Interchain Gas Payments, along with potential solutions.
+
+### `GasPaymentRequirementNotMet` Warning
+
+- **Reason:** This occurs when the gas payment provided during `dispatch` does not meet the relayer's calculated requirement.
+
+- **Solution:**
+  - Use `quoteGasPayment` to calculate the required payment for the destination domain and gas limit.
```

Comment:
ditto here, unfortunately we no longer want to point people to use this function because it does not accept the metadata overrides we reference above